pkgs/development/php-packages/amqp/default.nix

```diff
@@ -1,15 +1,28 @@
-{ buildPecl, lib, rabbitmq-c }:
+{ buildPecl, lib, rabbitmq-c, fetchFromGitHub, nix-update-script }:
 
-buildPecl {
-  pname = "amqp";
+let
+  version = "v1.11.0";
+in buildPecl {
+  inherit version;
+  pname = "apcu";
 
-  version = "1.11.0beta";
-  sha256 = "sha256-HbVLN6fg2htYZgAFw+IhYHP+XN8j7cTLG6S0YHHOC14=";
+  src = fetchFromGitHub {
+    owner = "php-amqp";
+    repo = "php-amqp";
+    rev = version;
+    sha256 = "sha256-CDhNDk78D15MtljbtyYj8euPnCruLZnc2NEHqXDX8HY=";
+  };
 
   buildInputs = [ rabbitmq-c ];
 
   AMQP_DIR = rabbitmq-c;
 
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = "php.extensions.amqp";
```

Comment:
This should no longer be necessary https://github.com/NixOS/nixpkgs/pull/207744

Reply:
Excellent ! I just updated all the commits and it works indeed. Thanks !!!